src/compute/mod.rs

```diff
@@ -103,7 +103,7 @@ fn compute_node_layout(
     // }
 
     // If this is a leaf node we can skip a lot of this function in some cases
-    let computed_size = if tree.children(node).is_empty() {
+    let computed_size = if tree.num_children(node) == 0 {
         #[cfg(feature = "debug")]
```

Comment:
Maybe a utility method similar to `is_empty` could make sense.
It could have a default implementation that just does `num_children(node) == 0`.
But maybe that would be unnecessary.

Reply:
Yes that's a good idea since it seems to be used a few times. I'm wondering if something like `is_childless` might be more descriptive. Since this abstracts over storage the idea of something being 'empty' to represent no children doesn't make a lot of sense to me, but then maybe I'm overthinking this.

Reply:
As always, naming is the hardest part of programming :D

Reply:
`is_childless` is nice to me.